packages/wp-now/src/wp-playground-wordpress/get-plugin-file.ts

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs-extra';
+import path, { basename } from 'path';
+
+/**
+ *
+ * @param filePath The path to the file to read.
+ * @param length The number of bytes to read from the file. By default 4KB.
+ * @returns The first `length` bytes of the file as string.
+ */
+function readFileHead(filePath: string, length = 4096) {
```

Comment:
In the core, we use [get_file_data()](https://developer.wordpress.org/reference/functions/get_file_data/) function for that and we read 8kb. What do you think about keeping those consistent?

Reply:
Awesome!, I increased the size to 8KB.
And I changed the regular expression to use the same in Core WordPress

https://github.com/WordPress/WordPress/blob/95bcde947f74dfb3cc2d57c624af5d904cf28b6c/wp-includes/functions.php#L6688